source/services/index.js

```diff
@@ -1,30 +1,33 @@
-'use strict';
-
 /**
  * Social network services
  */
+import Service from '../service';
+import { each } from '../utils';
+import svg from '../svg.js';
+import vkontakte from './vk';
 
-var Service = require('../service');
-var utils = require('../utils');
-var svg = require('../svg.js');
+import facebook from './facebook'; // eslint-disable-line sort-imports
```

Comment:
Let’s better disable the rule for the whole file than for a single line. This whole stuff that `eslint-disable-line` fixes the warning despite all the other imports are also sorted improperly looks like an ESLint bug.